docker-compose.yml

```diff
@@ -37,9 +40,9 @@ services:
     volumes:
       - ${MEDIA_ROOT}:/usr/local/app/src/media
     networks:
-      - network
+      - network-2024
```

Comment:
In order to align the network structure with [pycontw-nginx/docker-compose.yml](https://github.com/pycontw/pycontw-nginx/blob/main/docker-compose.yml).
I though it might be easier to understand.
```yml
networks:
    pycontw-net:
        external: true
    network-2024:
        external: true
```